.appveyor.yml

```diff
@@ -29,23 +35,23 @@ install:
         throw "There are newer queued builds for this pull request, failing early." }
 
   - "SET PATH=%PYTHON%;%PYTHON%\\Scripts;%PATH%"
-  - "python -m pip install --retries 3 -U pip"
+  - "python -m pip install -U pip"
 
   # Check that we have the expected version and architecture for Python
   - "python --version"
   - "python -c \"import struct; print(struct.calcsize('P') * 8)\""
   - "pip --version"
 
   # Install the build and runtime dependencies of the project.
-  - pip install %PIP_FLAGS% --retries 3 -r requirements/default.txt
-  - pip install %PIP_FLAGS% --retries 3 -r requirements/build.txt
+  - pip install %PIP_FLAGS% -r requirements/default.txt
+  - pip install %PIP_FLAGS% -r requirements/build.txt
   - python setup.py bdist_wheel bdist_wininst
   - ps: "ls dist"
 
   # Install the generated wheel package to test it.
   - pip install %PIP_FLAGS% --no-index --find-links dist/ scikit-image
   # Install the test dependencies
-  - pip install %PIP_FLAGS% --retries 3 -r requirements/test.txt
+  - pip install %PIP_FLAGS% -r requirements/test.txt
```

Comment:
Why are these removed?

Reply:
The [default now is 5](https://pip.pypa.io/en/stable/reference/pip/#cmdoption-retries) and I don't think the author wanted *less* retries